gen-cpp/CppRacconChat.h

```diff
@@ -0,0 +1,18 @@
+#include "RacconChat.h"
```

Comment:
use #pragma once: read more about it here https://en.wikipedia.org/wiki/Pragma_once